### ISSUE TITLE
After "Creating an Offering" with a meme, should flip the card back around

### ISSUE THREAD

**Issue by benzap** [district0x/memefactory#144]
Low Priority

How to reproduce
1. go to My Memefolio
2. go to Collected
3. click on a card that has been minted
4. Create an Offering
5. Pay the transaction
6. Upon finishing, the card will remain 'flipped' and remove the parameters. I believe the correct behaviour would be to 'unflip' the card upon finishing.

The Result after Step 6

![Image Broken](https://i.imgur.com/Xu9Mh96.png)

What I believe it should look like after Step 6

![Image Solution](https://i.imgur.com/oIOzrtK.png)

**Comment by jpmonettas** [comment]
@benzap I remember this was done on purpose so you can create multiple offerings of the same meme one after the other. 
@madvas what is the desired behavior here?

**Comment by madvas** [comment]
Can it turn back after submission, but when user clicks to turn again the form will stay filled out?

**Comment by jpmonettas** [comment]
@madvas Yes, that is the behavior after the fix
